pkgs/tools/security/der-ascii/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+let
+
+pname = "der-ascii";
+version = "0.1.0";
+
+in
+
+buildGoModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1my93m1rx08kn2yms6k8w43byr8k61r1nra4b082j8b393wwxkqc";
+  };
+  vendorSha256 = null;
+
+  ldflags = [ "-s" "-w" ];
```

Comment:
should we perhaps add a comment explaining what these ldflags are required for? curious because i was able to build the package without setting any.

Reply:
These are there in order to strip the executable; @azahi suggested them in a [previous comment](https://github.com/NixOS/nixpkgs/pull/215137#discussion_r1099313891) here and referenced the Go-wide (pan-Go?..) issue <https://github.com/NixOS/nixpkgs/issues/177698>. For my part, I’m not sure if this warrants a comment or not.

Reply:
No need for a comment. I'd suggest to leave it as is because we still haven't addressed the stripping issue fully.